include/mbedtls/mbedtls_config.h

```diff
@@ -3173,7 +3176,11 @@
  *
  * Module:  library/x509_create.c
  *
- * Requires: MBEDTLS_BIGNUM_C, MBEDTLS_OID_C, MBEDTLS_PK_WRITE_C
+ * Requires: MBEDTLS_BIGNUM_C, MBEDTLS_OID_C,
+ *           (MBEDTLS_MD_C or MBEDTLS_USE_PSA_CRYPTO)
+ *
+ * \warning If building without MBEDTLS_MD_C, you must call psa_crypto_init()
+ * before doing any PKCS#1 v2.1 operation.
```

Comment:
It seems to me that `x509_CREATE_C` does not really depend on MD - you probably meant `MBEDTLS_X509_CRT_WRITE_C` (but please make sure). Also, the description for `MBEDTLS_X509_CRT_WRITE_C` should end with 
```suggestion
 * before doing any x509 write operation.
```